.github/workflows/ci.yml

```diff
@@ -25,15 +25,16 @@ jobs:
           v0-${{ runner.os }}-pip-
     - name: Install dependencies
       run: |
-        python -m pip install --upgrade pip setuptools wheel
-        pip install .[lint]
+        # Pin pip to 21.x for now to avoid error in pre-commit action on Win
+        python -m pip install --upgrade pip==21.* setuptools wheel
```

Comment:
Is there a specific reason for `pip==21.*` vs `pip<22`? They both accomplished the same thing I just haven't seen the former used very often.

Reply:
It ensures you at least get the latest stable version of the 21.x branch, rather than any version below 22 (which could be a downgrade). In practice, there isn't really any difference (`pip~=21.3` would also be equivalent, and more precise, while `pip==21.3.1` would as well in practice, since per calver and pip policies 21.x won't get any more updates, barring extraordinary circumstances).

I'm happy to change it, if you prefer that spelling.

Reply:
Nope I'm good with it!